File: _BHCVRP_Python_Version/cujae_inf_citi_om/exceptions/ServiceTimeException.py
class ServiceTimeException(Exception):
    def __init__(self, service_time=None):
        if service_time is not None:
            self.service_time = service_time

    def __init__(self, message=None):
        if message is not None:
            super().__init__(message)

    def get_service_time(self):
        return getattr(self, "service_time", None)

File: BHCVRP_Java_Version/src/cujae/inf/citi/om/exceptions/ServiceTimeException.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package cujae.inf.citi.om.exceptions;

/**
 *
 * @author kmych
 */
public class ServiceTimeException extends Exception{
    private float serviceTime;
    
    public ServiceTimeException(float serviceTime){
        this.serviceTime = serviceTime;
    
    }
    
    public float getServiceTime(){
        return serviceTime;
    }
    
    public ServiceTimeException(String message){   
        super(message);
    }
    
}
